lib/apex/server.rb

```diff
@@ -61,9 +61,12 @@ def add_app_handlers
               # TODO, do the the rest of the types
               end
 
+           elsif request.raw.path.include?('.js')
```

Comment:
I did include because my script tags have stuff on the end after the .js 

Sent from my iPhone

> On 15 Apr 2015, at 00:21, Jamon Holmgren <notifications@github.com> wrote:
> 
> In lib/apex/server.rb:
> 
> > @@ -61,9 +61,12 @@ def add_app_handlers
> >                # TODO, do the the rest of the types
> >                end
> >  
> > +           elsif request.raw.path.include?('.js')
> Also, we probably want end_with? instead of include? here.
> 
> —
> Reply to this email directly or view it on GitHub.
>